core/claims/claimKeyBabyJub.go

```diff
@@ -0,0 +1,86 @@
+package claims
+
+import (
+	"encoding/binary"
+	"math/big"
+
+	"github.com/iden3/go-iden3-core/common"
+	"github.com/iden3/go-iden3-core/merkletree"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+)
+
```

Comment:
I would define the two documented KeyTypes as constants:
```go
const (
  BabyJubKeyTypeGeneric = 0
  BabyJubKeyTypeAuthorizeKSign = 1
)
```

I would also suggest defining a new type for the KeyType, but I leave this up to you.  The advantages of this would be that it would be straight forward to relate the defined constants with the arguments of the NewClaimKeyBabyJub because they would be the same type.

```go
type BabyJubKeyType uint64
```

Reply:
Cool, done in that way